static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -185,6 +185,7 @@ define([
             embedPath = $el.attr('data-embed-path'),
             // we need to look up the embedPath for main media videos
             canonicalUrl = $el.attr('data-canonical-url') || (embedPath ? '/' + embedPath : null),
+            shouldHideAdverts = $el.attr('data-block-video-ads') === 'false' ? false : true,
```

Comment:
this value isn't always correct, as embeds don't follow parent updates 😢 I guess its a good default value though...